### PR TITLE
Copied modified versions of dolt docs into doltgres

### PR DESCRIPTION
**Pull request by zachmu**
Probably a lot of links and images are broken, will do a second pass for those.

### CODE REVIEW

File: packages/doltgres/content/reference/sql/benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/benchmarks/README.md]: This is already there.

File: packages/doltgres/content/reference/sql/benchmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+to other relational databases. Additional benchmarks will be added
+over time.
+
+To learn more, click on a subsection heading.
+
+* [SQL correctness](./correctness.md) tests Dolt's query engine against
+  the `sqllogictest` suite for correctness issues.
+* [Server latency](./latency.md) uses `sysbench` to compare Dolt's read
+  and write latencies against MySQL.
+* [Import latency](./import.md) uses a custom benchmark to compare Dolt's bulk import performance
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/benchmarks/README.md]: Kill this.

File: packages/doltgres/content/reference/sql/benchmarks/correctness.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/benchmarks/correctness.md]: Kill this.

File: packages/doltgres/content/reference/sql/benchmarks/import.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/benchmarks/import.md]: Kill

File: packages/doltgres/content/reference/sql/benchmarks/latency.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/benchmarks/latency.md]: Kill

File: packages/doltgres/content/reference/sql/server/metrics.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/server/metrics.md:1]: I don't think this is true yet.

File: packages/doltgres/content/reference/sql/server/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+title: Troubleshooting
+---
+
+Debugging a running Doltgres server can be challenging. This document covers the debugging basics and how to diagnose what is happening from common symptoms.
+
+# Basics
+
+## Make sure you are running the latest Doltgres version
+
+Doltgres is constantly evolving. We release a new Doltgres approximately once a week. Connect to the SQL server and run `select dolt_version()`. Make sure the version matches the latest as seen on the [GitHub releases page](https://github.com/dolthub/dolt/releases).
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/server/troubleshooting.md]: is it really dolt_version()?

File: packages/doltgres/content/reference/sql/server/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+To upgrade the server, download the latest Doltgres binary for your platform and replace the Doltgres binary on your `PATH` with the downloaded one. Running the install process on most platforms again will do this for you. Restart the Doltgres server using `dolt sql-server` to have your running server start using the latest binary.
+
+## Examine your CPU, Memory, and Disk usage
+
+Doltgres consumes CPU, Memory, and Disk. Consuming more of any of these resources than the host has available can lead to degraded performance. Use your system's built in resource monitoring systems to inspect Doltgres's usage of these resources. You may need a larger host or additional [read replicas](./replication.md) to support your load.
+
+## Set your log level to DEBUG or TRACE
+
+To see queries being run against the server, query results, and query latency set your Doltgres log level to `DEBUG` or `TRACE`. This can be done by starting the server like so `dolt sql-server --loglevel=debug` or by setting `log_level: debug` in your `config.yaml`. Your logs should be visible in the shell you started `dolt sql-server` in.
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/server/troubleshooting.md]: dolt

File: packages/doltgres/content/reference/sql/server/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+## EXPLAIN for complex queries
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/server/troubleshooting.md]: I don't think EXPLAIN works yet

File: packages/doltgres/content/reference/sql/server/troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+Doltgres supports the SQL `EXPLAIN` operation in order for you to see the plan for complex queries. Rearranging your query to perform fewer `JOIN`s or make better use of indexes can help speed up complex queries.
+
+## Compare to MySQL
+
+Doltgres strives to be 100% MySQL compatible. If you run a query that works in MySQL but does not work in Doltgres, it is a Doltgres bug and you should [submit an issue](#submitting-issues). You can dump your Doltgres database using [`dolt dump`](../../cli/cli.md#dolt-dump) and import the resulting file into MySQL using `mysql < dump.sql`. The test the query you think should work using any MySQL client.
```
Comment (timsehn) [inline packages/doltgres/content/reference/sql/server/troubleshooting.md]: Postgres

**Review by timsehn — COMMENTED** [review]
Found some issue. Probably best to debug next step once it is published on dev.

File: packages/doltgres/content/reference/sql/server/troubleshooting.md (hunk 7 above)
Comment (zachmu) [reply packages/doltgres/content/reference/sql/server/troubleshooting.md]: Yeah, pretty sure. I'll add an issue.

File: packages/doltgres/content/reference/sql/server/metrics.md (hunk 6 above)
Comment (zachmu) [reply packages/doltgres/content/reference/sql/server/metrics.md:1]: I think it works, I don't know why it wouldn't

**Pull request merged**